rasa/core/featurizers/single_state_featurizer.py

```diff
@@ -354,10 +354,33 @@ def _encode_action(
 
         return self._extract_state_features(action_as_sub_state, interpreter)
 
+    def encode_all_labels(
```

Comment:
Just to check -- this is really just a re-naming of the old method (encode_all_actions)? Was this to be more accurate/descriptive?

Reply:
Yes, it's a renaming for the function inside `SingleStateFeaturizer`, so that a common function (encode_all_labels) can be invoked from the policy and not an implementation specific function (encode_all_intents / encode_all_actions) is invoked